### PR TITLE
Updated download page layout and content

### PR DESCRIPTION
**Pull request by WilliamHarvey97**

### CODE REVIEW

File: web/src/views/analyze_sleep/waiting_for_server/index.js [hunk 1]
```diff
@@ -11,27 +11,31 @@ const WaitingForServer = () => {
   }, [setAssets]);
 
   return (
-    <Container>
-      <h3 className="mb-4">Waiting for local server to be running...</h3>
+    <Container className="my-6">
+      <Alert color="warning" className="lead mb-5 mr-3">
+        <i class="fas fa-server fa-lg mr-3" />
+        <strong>Waiting for local server to be running...</strong>
```
Comment (abelfodil) [inline web/src/views/analyze_sleep/waiting_for_server/index.js:17]: Maybe add something like this to the text?

File: web/src/views/analyze_sleep/waiting_for_server/index.js [hunk 2]
```diff
@@ -11,27 +11,31 @@ const WaitingForServer = () => {
...
+            In order to upload your sleep file, we ask you to install our local server. This server will proceed to the
+            preprocessing, feature extraction and classification of your sleep EEG. Your data is processed locally since
+            we, at PolyCortex, believe that you should have full control over your data and its privacy.
+          </p>
+        </Col>
+        <Col md="6" className="text-center mb-4">
           <ListGroup>
             {assets.map((asset) => (
-              <ListGroupItem key={asset.name} active={asset.currentOs} tag="a" href={asset.url}>
+              <ListGroupItem color="default" key={asset.name} active={asset.currentOs} tag="a" href={asset.url}>
```
Comment (abelfodil) [inline web/src/views/analyze_sleep/waiting_for_server/index.js:32]: perso j'aime pas la couleur, ça clash un peu

**Review by abelfodil — CHANGES_REQUESTED** [review]

File: web/src/views/analyze_sleep/waiting_for_server/index.js (hunk 1 above)
Comment (WilliamHarvey97) [reply web/src/views/analyze_sleep/waiting_for_server/index.js:17]: Like what :interrobang:

Comment (abelfodil) [reply web/src/views/analyze_sleep/waiting_for_server/index.js:17]: oops bad copy pasta

here's the link: https://stackoverflow.com/questions/4639716/dot-dotdot-dotdotdot-as-loading

**Review by abelfodil — APPROVED** [review]

**Pull request merged**